applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -104,9 +104,9 @@ def _solve(self,mp):
 
         #define a minimal newton raphson solver
         linear_solver = KratosMultiphysics.SkylineLUFactorizationSolver()
-        builder_and_solver = KratosMultiphysics.ResidualBasedEliminationBuilderAndSolver(linear_solver)
+        builder_and_solver = KratosMultiphysics.ResidualBasedBlockBuilderAndSolver(linear_solver)
```

Comment:
do you really want to change this?
I would leave it, since otherwise we have almost no tests using it
The BlockBuilder is used everywhere

Reply:
well, maybe i can modify the test to use one or the other. point is that i ultimately want to use this as a test also for the future PR on using the old stiffness at first iteration

Reply:
+1 for using both